src/chromo/models/dpmjetIII.py

```diff
@@ -225,6 +236,11 @@ def _generate(self):
         self._lib.dtevno.nevent += 1
         return not reject
 
+    def print_original_event(self, mode=1):
+        if hasattr(self._lib, "dtflka"):
+            self._lib.dtflka.lpri = 5
```

Comment:
Bug.  Lpri state should be saved and then restored after the fit.

Reply:
Fixed